lists/bo/bo-nit.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": {
+    "en": "Tax Identification Number",
+    "local": "Número de Identificación Tributaria "
+  },
+  "coverage": [
+    "BO"
+  ],
+  "description": {
+    "en": "The NIT or Identify Tax Number is the number assigned by the Tax Administration to a natural person, a legal entity at the time of registration for any economic activity, by virtue of which they are taxable persons of some of the taxes established by Law.\nhttps://impuestos.com.bo/rnd-03-13/"
```

Comment:
Could be edited a little for clarity. Suggestion:

"The _Número de Identificación Tributaria (NIT)_, or Tax Identification Number, is the number assigned by the National Tax Service of Bolivia to a taxable natural person or legal entity. It is assigned once an individual or entity registers with the Service, acknowledging that their economic activity can, by law, be taxed.\nhttps://impuestos.com.bo/rnd-03-13/"